src/Installer.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace OpenCensus\Trace\Exporter;
+
+class Installer
+{
+    public static function checkPhpExtDependency()
+    {
+        if (!function_exists('bcadd') && !function_exists('gmp_add')) {
```

Comment:
Let's use `extension_loaded` here - a user *could* have `bcadd` or `gmp_add` implemented somewhere in the function lookup chain and not the rest.